static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -1,69 +1,108 @@
+import {useTheme} from '@emotion/react';
 import styled from '@emotion/styled';
 
 import {AreaChart, AreaChartProps} from 'sentry/components/charts/areaChart';
+import Grid from 'sentry/components/charts/components/grid';
+import Tooltip from 'sentry/components/charts/components/tooltip';
+import XAxis from 'sentry/components/charts/components/xAxis';
+import YAxis from 'sentry/components/charts/components/yAxis';
 import {MemorySpanType} from 'sentry/components/events/interfaces/spans/types';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {formatBytesBase2} from 'sentry/utils';
 import {getFormattedDate} from 'sentry/utils/dates';
-import theme from 'sentry/utils/theme';
 import EmptyMessage from 'sentry/views/settings/components/emptyMessage';
 
 type Props = {
   memorySpans: MemorySpanType[] | undefined;
+  startTimestamp: number | undefined;
 };
 
 const formatTimestamp = timestamp =>
-  getFormattedDate(timestamp * 1000, 'MMM D, YYYY HH:mm:ss');
+  getFormattedDate(timestamp * 1000, 'MMM D, YYYY hh:mm:ss A', {local: false}).concat(
+    ' UTC'
+  );
```

Comment:
I think `z` inside the format string will work to replace the `.concat(' UTC')` call.

https://momentjs.com/docs/#/displaying/

```suggestion
  getFormattedDate(timestamp * 1000, 'MMM D, YYYY hh:mm:ss A z', {local: false});
```